storybook/package.json

```diff
@@ -36,5 +37,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "webpack-cli": "^3.3.11"
```

Comment:
Why do you have to install this for multiple folders?

Reply:
aaahhh I did it through Lerna and didn't hoist.